packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -453,7 +453,10 @@ export const NATIVE_FEE_TOKEN_BY_CHAIN_ID = valueByChainId<string>(
     NULL_ADDRESS,
 );
 
-export const NATIVE_FEE_TOKEN_AMOUNT_BY_CHAIN_ID = valueByChainId({}, ONE_ETHER);
+export const NATIVE_FEE_TOKEN_AMOUNT_BY_CHAIN_ID = valueByChainId(
+    { [ChainId.Mainnet]: ONE_ETHER.times(0.1) },
```

Comment:
?

Reply:
Oh yeah, I felt like getting querying $4000 of the token value might be a bit high these days. WDYT

Reply:
Yeah seems fair